src/ios/CDVCamera.m

```diff
@@ -512,8 +512,13 @@ - (void)resultForImage:(CDVPictureOptions*)options info:(NSDictionary*)info comp
 
 - (CDVPluginResult*)resultForVideo:(NSDictionary*)info
 {
-    NSString* moviePath = [[info objectForKey:UIImagePickerControllerMediaURL] absoluteString];
-    return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:moviePath];
+    @try {
+        NSString* moviePath = [[info objectForKey:UIImagePickerControllerMediaURL] absoluteString];
+        return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:moviePath];
+    } @catch (NSException *exception) {
+        NSLog(@"Camera.resultForVideo: error retrieving file path");
+        return [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@""];
```

Comment:
Great, one minor request: Could you pass in the exception message into the Cordova callback?

Reply:
Sorry for the late response, I left the company I was working in at the time and lost the email account, and also access to this github account. All should be fine now